Mod2/Lessons/Week1/SQLCRUD.md

```diff
@@ -0,0 +1,132 @@
+# SQL CRUD
+
+## Learning Goals
+- Define CRUD
+- Use SQL to SELECT, INSERT, REMOVE and UPDATE records
+- Use basic WHERE clause to filter records
+
+## Warmup
+
+Spend about 10 minutes googling `CRUD` (all caps) as it relates to programming, databases, and SQL.
+* What does each letter stand for?
+* In your own words, what does each action do?
+
+Be ready to share your findings!
+
+**For this lesson, all SQL examples will be based on our Orders and Customers tables that we created in the [Intro to Databases Lesson](/Mod2/Lessons/Week1/IntroToDatabases.md)!**
+
+## Inserting Data in our Tables
+
+In SQL, we **create** records using the `INSERT INTO` command.  `INSERT INTO` can add single or multiple records.
+
+> With a partner, take a look at the two insert statements below and answer the following questions:
+> * What do we need to know about a table in order to create records?
+> * What do you think would happen if we did not include a customer's address?
+> * Why do you not see an id being created?
```

Comment:
I would add a note encouraging students to also try running different inserts and see what happens. 

I also like the idea of moving this line up so that if students are trying things out they have a way to check what's happening.

To check that records have been created, or anytime you want to see the current state of a table, you can run this statement: `SELECT * FROM <tablename>;`